c++/defs/dependencies.bzl

```diff
@@ -0,0 +1,29 @@
+"Provides dependency version mappings for Cap'n'Proto and downstream Bazel projects."
+
+
+## Brotli version to depend on.
+BROTLI = struct(
+    version = "1.1.0",
+    digest = "e720a6ca29428b803f4ad165371771f5398faba397edf6778837a18599ea13ff",
+)
+
+## Zlib version to depend on.
+ZLIB = struct(
+    version = "1.3",
+    digest = "8a9ba2898e1d0d774eca6ba5b4627a11e5588ba85c8851336eb38de4683050a7",
+)
```

Comment:
Dependencies for `WORKSPACE`-based builds are now defined here, and should be updated here, in sync with `MODULE.bazel`. Unfortunately there is no way to import from `MODULE.bazel` even for simple symbols, so this minimal duplication must exist.